<a href="https://colab.research.google.com/github/LuisaMariaMontoyaConde1994/TrabajoClaseestadistica/blob/main/Actividad_1_Daniela_Gualtero_M.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
import numpy as np
import matplotlib.pyplot as plt
from scipy.stats import binom


En una elección, un candidato tiene una probabilidad del $60\%$ de recibir un voto en cada votación individual. Si se emiten 20 votos.


1.  ¿Cuál es la probabilidad de que al menos obtenga $12$ votos?
2.  ¿Cuál es la probabilidad de que lo más obtenga $5$ votos?
3.  ¿Cuál es la probabilidd de que no obtenga votos.
4.  ¿Cuál es la probabilidad de que obtenga entre $8$ y $10$ votos?

1. ¿Cuál es la probabilidad que al menos obtenga $12$ votos?
$$P(X\geq 12)=1-P(X< 12)$$

In [2]:
n=20
p=0.6
x=12
prob=1-binom.cdf(x-1,n,p)
m=prob*100
print('La probabilidad es',m,'%')

La probabilidad es 59.55987253122248 %


La probabilidad de que al menos obtenga $12$ votos es del $59.55\%$

2.   ¿Cuál es la probabilidad de que lo más obtenga $5$ votos?
$$p(X\leq 5)$$

In [4]:
n=20
p=0.6
x=5
prob=binom.cdf(x,n,p)
print(prob)
m=prob*100
print('La probabilidad es',m,'%')

0.001611524643456288
La probabilidad es 0.1611524643456288 %


La probabilidad de que lo más obtenga $5$ votos es del $0.16\%$

3.  ¿cual es la probabilidad que no obtenga votos?
$$P(x=0)=\binom{20}{0}(0.6)^{0}(0.4)^{20-0}$$
$$\approx 1.0995116277760013e-08 $$
$$\approx 1.0995116277760013e-08(100\%)=1.09e\%$$


In [20]:
n=20
p=0.6
x=0
prob=binom.pmf(x,n,p)
print(prob)
m=prob*100
print('la probabilidad es',m,'%')

1.0995116277760013e-08
la probabilidad es 1.0995116277760013e-06 %


La probabilidad de que no obtenga votos es de $1.0995116277760013e-06 %$

4.  ¿Cuál es la probabilidad de que obtenga entre $8$ y $10$ votos?


In [16]:
n=20
p=0.6
x=10
prob=binom.cdf(x,n,p)
print(prob)
r=prob*100
print('La probabilidad es',r,'%')

0.24466279668360685
La probabilidad es 24.466279668360684 %


In [17]:
n=20
p=0.6
x=8
prob=binom.cdf(x,n,p)
print(prob)
t=prob*100
print('La probabilidad es',t,'%')

0.056526367034253025
La probabilidad es 5.652636703425302 %


In [19]:
p=r-t
print(p)

18.813642964935383


La probabilidad de que obtenga entre 8 y 10 votos es de $18.81%$